css/style.css

```diff
@@ -1,4 +1,503 @@
 .body {
   font-family: 'Montserrat', sans-serif;
   font-weight: normal;
+  max-width: 1440px;
+  min-width: 320px;
+  overflow-x: hidden;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  box-sizing: border-box; 
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+  padding: 21px 5px 74px 0px;
+}
+
+.header__logo-img {
+  width: 163px;
+  height: 48px;
+}
+.menubar {
+  display: flex;
+  align-items: center;
+}
+
+.menubar__link {
+  text-decoration: none;
+  font-weight: 600;
+  font-size: 15px;
+  color: black;
+  white-space: nowrap;
+}
+
+.menubar__link:hover, 
+.menubar__link:active, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)